package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-ledger",
-  "version": "8.8.1",
+  "version": "8.8.2",
```

Comment:
8.8.1 is still fine, we don't have a release for that yet

Reply:
ok cool. I've changed it back.